odoo/addons/base/tests/asset_tag.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <asset id="base.test_asset_tag_aaa" name="Test asset tag (init active => keep active => update active)">
+        <bundle>test_asset_bundle</bundle>
+        <path>base/tests/something.scss</path>
+    </asset>
+    <asset id="base.test_asset_tag_aii" name="Test asset tag (init active => make inactive => update inactive)">
```

Comment:
Triple check how are fake/tests data done in base module, probably to be moved too, didn't check